dashboard/db/migrate/20220525175523_create_code_reviews.rb

```diff
@@ -0,0 +1,24 @@
+class CreateCodeReviews < ActiveRecord::Migration[6.0]
+  def change
+    create_table :code_reviews do |t|
```

Comment:
Should we also drop the old code_review_requests table as part of this?

Reply:
I need to drop the old table separately, after the model changes have been deployed.